File: Modules/HumanResource/TrainingManagement/ProgramFlow/ProgramFlowType.cs
using System.ComponentModel;
using Serenity.Data;

namespace HRMSoftware.TrainingManagement;
[Serenity.ComponentModel.EnumKey("HumanResource.TrainingManagement.TrainingProgramFlowType")]
public enum ProgramFlowType : int
{
    [Description("None")]
    None = 0,
    [Description("Attendance")]
    Attendance = 1,
    [Description("Submit Document")]
    Document = 2,
    [Description("Grading")]
    Assessment = 3,
}

File: Modules/ServerTypes/TrainingManagement/ProgramFlowType.ts
import { Decorators } from "@serenity-is/corelib";

export enum ProgramFlowType {
    None = 0,
    Attendance = 1,
    Document = 2,
    Assessment = 3
}
Decorators.registerEnumType(ProgramFlowType, 'HRMSoftware.TrainingManagement.ProgramFlowType', 'HumanResource.TrainingManagement.TrainingProgramFlowType');